src/utils/stellar.ts

```diff
@@ -74,10 +74,10 @@ class StellarCustodial {
   public async makePayment(source: string, dest: string, amount: string): Promise<any> {
     const sourceMuxed = this.muxedFromAddress(source);
     const sourceUser = await getUserById(sourceMuxed.id());
-
+    console.log(sourceUser);
     if (!sourceUser) throw new Error('source-not-found');
-
-    if (sourceUser.balance < amount) throw new Error("insufficient-balance")
+    console.log(sourceUser.balance, amount, sourceUser.balance < amount)
+    if (parseFloat(sourceUser.balance) < parseFloat(amount)) throw new Error("insufficient-balance")
```

Comment:
please don't use floats 
```suggestion
    if (new BigNumber(sourceUser.balance).lt(new BigNumber(amount))) throw new Error("insufficient-balance")
```

Reply:
What if amount has decimal?

Reply:
Then the problems start to kick in: https://stackoverflow.com/a/3730040/834309

Reply:
https://developers.stellar.org/docs/issuing-assets/anatomy-of-an-asset/

Reply:
By BigNumber, you mean this library, right? https://mikemcl.github.io/bignumber.js/

Reply:
exactly